docs/core/diagnostics/distributed-tracing-concepts.md

```diff
@@ -0,0 +1,137 @@
+ID scheme is used. In the W3C TraceContext standard every trace is assigned a globally unique 16
+byte trace-id (<xref:System.Diagnostics.Activity.TraceId?displayProperty=nameWithType>) and
+every Activity within the trace is assigned a unique 8 byte span-id
+(<xref:System.Diagnostics.Activity.SpanId?displayProperty=nameWithType>). Each Activity
+records the trace-id, its own span-id, and the span-id of its parent
+(<xref:System.Diagnostics.Activity.ParentSpanId?displayProperty=nameWithType>). Because
+distributed traces can track work across process boundaries parent and child Activities may
+not be in the same process. The combination of a trace-id and parent span-id can uniquely
+identify the parent Activity globally, regardless of what process it resides in.
+
+<xref:System.Diagnostics.Activity.DefaultIdFormat?displayProperty=nameWithType> controls which
+ID format is used for starting new traces, but by default adding a new Activity to an existing
+trace uses whatever format the parent Activity is using.
+Setting <xref:System.Diagnostics.Activity.ForceDefaultIdFormat?displayProperty=nameWithType>
+to true overrides this behavior and creates all new Activities with the DefaultIdFormat, even
+when the parent uses a different ID format.
+
+### Starting and stopping Activities
+
+Each thread in a process may have a corresponding Activity object that is tracking the work
+occuring on that thread, accessible via
+<xref:System.Diagnostics.Activity.Current?displayProperty=nameWithType>. The current activity
+automatically flows along all synchronous calls on a thread as well as following async calls
+that are processed on different threads. If Activity A is the current activity on a thread and
+code starts a new Activity B then B becomes the new current activity on that thread. By default
+activity B will also treat Activity A as its parent. When Activity B is later stopped activity
+A will be restored as the current Activity on the thread. When an Activity is started it
+captures the current time as the
+<xref:System.Diagnostics.Activity.StartTimeUtc?displayProperty=nameWithType>. When it
+stops <xref:System.Diagnostics.Activity.Duration?displayProperty=nameWithType> is calculated
+as the difference between the current time and the start time.
+
+### Coordinating across process boundaries
+
+In order to track work across process boundaries Activity parent IDs need to be transmitted across
+the network so that the receiving process can create Activities that refer to them. When using
+W3C TraceContext ID format .NET will also use the HTTP headers recommended by
+[the standard](https://www.w3.org/TR/trace-context/) to transmit this information. When using the
+<xref:System.Diagnostics.ActivityIdFormat.Hierarchical> ID format
+.NET uses a custom request-id HTTP header to transmit the ID. Unlike many other language runtimes
+.NET in-box libraries natively understand how to decode and encode Activity IDs on HTTP messages
```

Comment:
might be a good idea to clarify what is "in-box" libraries. You are referring to ASP.NET Core and HttpClient in .NET Core right?